lib/bootstrap_form/components/labels.rb

```diff
@@ -25,13 +25,7 @@ def generate_label(id, name, options, custom_label_col, group_layout)
       def label_classes(name, options, custom_label_col, group_layout)
         classes = ["form-label", options[:class], label_layout_classes(custom_label_col, group_layout)]
 
-        case options.delete(:required)
-        when true
-          classes << "required"
-        when nil, :default
-          classes << "required" if required_attribute?(object, name)
-        end
-
+        classes << "required" if options.delete(:required)
```

Comment:
Just looking at this change, it looks like a change to the behaviour of the software. Why don't we care about the `nil, :default` case anymore?